integration_test/dapp_tests/constants.js

```diff
@@ -1,16 +1,25 @@
 const rpcUrls = {
+    "seilocal": "http://127.0.0.1:26657",
     "testnet": "https://rpc-testnet.sei-apis.com",
-    "devnet": "https://rpc-arctic-1.sei-apis.com"
+    "devnet": "https://rpc-arctic-1.sei-apis.com",
+    "devnetFastTrack": "https://rpc-arctic-1.sei-apis.com",
```

Comment:
can `devnetFastTrack` just use the same rpc as `devnet`?

Reply:
Yes, updated.